lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -508,23 +537,34 @@ class IOSTextEditingStrategy extends DefaultTextEditingStrategy {
     domElement.style.transform = 'translate(-9999px, -9999px)';
 
     _canPosition = false;
+  }
+
+  @override
+  void addEventHandlers() {
+    // Subscribe to text and selection changes.
+    _subscriptions.add(domElement.onInput.listen(_handleChange));
+
+    _subscriptions.add(domElement.onKeyDown.listen(_maybeSendAction));
 
+    _subscriptions.add(html.document.onSelectionChange.listen(_handleChange));
+
+    /// Position the DOM element after it is focused.
```

Comment:
nit: `//` (`///` is used for dartdocs only)

Reply:
done